specification/semantic_conventions.md

```diff
@@ -99,7 +99,16 @@ For each `InstrumentationLibraryLogs` instance:
 - `name` - MUST be set to `otel.profiling`
 - `version` - MUST be set to `0.1.0`
 
-The resource field in `ResourceLogs` MUST contain the [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk) attribute.
+The resource field in `ResourceLogs`:
+
+- MUST contain following attributes:
+  - [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk)
+  - [`service.name`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service)
```

Comment:
the OTel SDKs are always adding (it is required by OTel spec), what is more we also have this: https://github.com/lachmatt/gdi-specification/blob/0a642e7574b0dd964da4ae157a4a34f8d1d3d3dc/specification/configuration.md#L203-L213

Reply:
removed in https://github.com/signalfx/gdi-specification/pull/206/commits/dba1bff7c947c20ad56fab6607df000f370fd80c